wallet/src/account.rs

```diff
@@ -0,0 +1,51 @@
+use crate::types;
+
+/// Result type for accounts-related operations that can fail.
+pub type Result<T> = std::result::Result<T, failure::Error>;
+
+/// Root KeyPath used for wallet accounts.
+#[inline]
+pub fn account_keypath(index: u32) -> types::KeyPath {
```

Comment:
Perhaps we should add a link to the WIP: https://github.com/aesedepece/WIPs/blob/wip-adansdpc-hdwallets/wip-adansdpc-hdwallets.md

Reply:
I'll add this https://github.com/aesedepece/WIPs/blob/wip-adansdpc-hdwallets/wip-adansdpc-hdwallets.md#path-levels